app/views/trainees/course_details/edit.html.erb

```diff
@@ -58,4 +58,4 @@
   </div>
 </div>
 
-<p class="govuk-body"><%= govuk_link_to("Cancel", view_trainee(@trainee)) %></p>
+<p class="govuk-body"><%= govuk_link_to(t("cancel"), view_trainee(@trainee)) %></p>
```

Comment:
Should the key in the en.yml be `cancel_and_return_to_record` like the other one given that is has the same value?